internal/fs/utils.go

```diff
@@ -37,6 +37,25 @@ func HasCExtension(file string) bool {
 	return withExtension(".c", ".cc", ".cxx", ".cpp")(file)
 }
 
+// returns the path of the directory containing k3.env
```

Comment:
```suggestion
// FindK3EnvInCurrPath returns the path of the directory containing k3.env
```

Reply:
The name FindK3EnvIn**Curr**Path is slightly misleading, because this function does not check the current path, but the one passed as parameter.